nohost/src/HTMLServer.js

```diff
@@ -76,18 +76,9 @@ define(function (require, exports, module) {
     HTMLServer.prototype.serveLiveDoc = function(url, callback) {
         var path = BlobUtils.getFilename(url);
         var liveDocument = this._liveDocuments[path];
+        var rewrittenHtml = Rewriter.rewrite(path, liveDocument.getResponseData().body);
```

Comment:
How can this be synchronous?  Where did you make the change to `rewriter` to make that possible?